db/migrate/20201104115520_set_internal_sequence_value.rb

```diff
@@ -0,0 +1,16 @@
+class SetInternalSequenceValue < ActiveRecord::Migration[5.2]
+  class OrderProcess < ActiveRecord::Base
+  end
```

Comment:
We want to have a simple model so that we can bypass tenant, before callbacks etc. It is also a best practice not to use the app's default model http://vesavanska.com/2014/activerecord-migrations-best-practices

Reply:
Ok, that makes sense. Thanks 👍